<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weather</class>
 <widget class="QWidget" name="Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="weather" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#weather{
	border-image: url(:/imgs/weather_bg.png);
}</string>
   </property>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imgs/back.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dateLab">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2025-5-8</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>10:20:10  周三</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cityLab">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 30pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>四川</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>72</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imgs/local.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tLab">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 30pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>27℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ticon">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dayLab">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>晴天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
color: rgb(253, 254, 174);
</string>
    </property>
    <property name="text">
     <string>2025-05-07
周六</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tLab1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(253, 254, 174);</string>
    </property>
    <property name="text">
     <string>20℃~27℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ticon1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dayLab1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
color: rgb(253, 254, 174);</string>
    </property>
    <property name="text">
     <string>晴天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>2025-05-07
周六</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>2025-05-07
周六</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>2025-05-07
周六</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>2025-05-07
周六</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>2025-05-07
周六</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>280</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>2025-05-07
周六</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tLab2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>20℃~27℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tLab3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>20℃~27℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tLab4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>330</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>20℃~27℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tLab5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>20℃~27℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tLab6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>20℃~27℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tLab7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>20℃~27℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ticon2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ticon3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ticon4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ticon5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ticon6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ticon7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dayLab2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>晴天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dayLab3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>晴天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dayLab4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>晴天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dayLab5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>460</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>晴天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dayLab6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>晴天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dayLab7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>460</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;

color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>晴天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
